developers/verifiers/index.md

```diff
@@ -27,13 +29,13 @@ await credentialHandlerPolyfill.loadOnce();
 
 Or, if you're developing on Node.js, add the credential-handler-polyfill library to your project...
 
-```
+```sh
 npm i credential-handler-polyfill@3.0.1
```

Comment:
@davidlehn @dlongley do we really want to specify a polyfill version here?

Reply:
There are other places with specific versions too.  I'm not sure what best practice is for this without invoking complex tooling. For npm it's probably not too bad and would pick up newer semantic version.  But for the script tags with explicit versions, they will be locked at older version.  If nothing else, could add short notes to check for newer versions and link to the releases page.

Reply:
We should probably not specify a version -- provided that we remember to keep these docs up to date when we make any breaking changes.